youtube_dlc/extractor/niconico.py

```diff
@@ -386,8 +385,7 @@ def get_video_info_xml(items):
                 raise ExtractorError('The video is not found.',
                                      expected=True)
             elif error_code == 'COMMUNITY':
-                raise ExtractorError('The video is community members only.',
-                                     expected=True)
+                self.to_screen('%s: The video is community members only.' % video_id)
```

Comment:
Why `to_screen`? Can it still be downloaded?

Reply:
Yes. Community members can't download community members only movies.
But, raise ExtractorError another place for other users.